### PR TITLE
Ability to spacify specific words of a text

### PR DESCRIPTION
**Pull request by Arthur-agl**
Surround the text to be spacified with  ., ,.

### CODE REVIEW

**Comment by Arthur-agl** [comment]
- Spacify all the text if no tag is provided
- Spacify all the text after the tag if only one tag is provided

File: api/index.js [hunk 1]
```diff
@@ -2,30 +2,31 @@ const url = 'https://api.telegram.org/bot';
 const apiToken = process.env.BOT_TOKEN;
 const axios = require('axios');
 const crypto = require('crypto');
+const space = require('./spacifier');
```
Comment (jgobi) [inline api/index.js]: I think it's misspelled (see reference in line 11)
```suggestion
const spacify = require('./spacifier');
```

File: spacifier.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const spacify = (str) => {
+	let splitResult = str.split('..');
```
Comment (jgobi) [inline spacifier.js]: If you want to use `,.` or `.,` separator interchangeable

```suggestion
	let splitResult = str.split(/\.,|,\./);
```

File: spacifier.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+	if (splitResult.length === 1) return splitResult[0].split('').join(' ');
```
Comment (jgobi) [inline spacifier.js]: maybe u can turn `string.split('').join(' ')` in a function as it's used twice

**Review by jgobi — CHANGES_REQUESTED** [review]
Thanks for your PR! I'll be glad to merge it!

Although I preffer space over tabs for identing, the previous version was made in a hurry and have mixed both in this project. As you have changed the identation to tabs, I'll mantain this :)

I do preffer the latter separator `.,` or `,.` (can be both interchangeable too, see my suggestion if you want), as I can expect this output from the bot: "I think it'll by  o k a y....", but you can choose.

Please just see my code review and I'll do the merge and deploy the bot =)

File: api/index.js (hunk 1 above)
Comment (Arthur-agl) [reply api/index.js]: Oh thanks for pointing out!

File: spacifier.js (hunk 2 above)
Comment (Arthur-agl) [reply spacifier.js]: By the power of democracy we'll keep using .. for now

File: spacifier.js (hunk 3 above)
Comment (Arthur-agl) [reply spacifier.js]: Changed because I also like them spacez

File: spacifier.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+function _spacify(str){ return str.split('').join(' ');}
```
Comment (jgobi) [inline spacifier.js]: please be consistent in function declaration

```suggestion
const _spacify = str => str.split('').join(' ');
```

**Review by jgobi — CHANGES_REQUESTED** [review]
thanks for changing tabs to space =)

File: api/index.js [hunk 5]
```diff
@@ -2,30 +2,31 @@ const url = 'https://api.telegram.org/bot';
...
                     res.status(200).send(response);
-               }).catch((error) => {
+        }).catch((error) => {
                     res.send(error);
-               });
+                });
      } else {
-          res.status(200).send({});
+            res.status(200).send({});
      }
```
Comment (jgobi) [inline api/index.js]: sorry for being pedantic, but can you please fix this identation?

**Review by jgobi — CHANGES_REQUESTED** [review]

File: spacifier.js (hunk 2 above)
Comment (jgobi) [reply spacifier.js]: Alright

**Review by jgobi — APPROVED** [review]
thanks for the PR. I'll deploy soon

**Pull request merged**